src/mapnode.cpp

```diff
@@ -215,7 +215,7 @@ void MapNode::rotateAlongYAxis(INodeDefManager *nodemgr, Rotation rot)
 }
 
 static std::vector<aabb3f> transformNodeBox(const MapNode &n,
-		const NodeBox &nodebox, INodeDefManager *nodemgr)
+		const NodeBox &nodebox, INodeDefManager *nodemgr, u8 neighbors)
```

Comment:
Can you default the param to 0 here too? it would spare some changes where only 0 is added to the call of `transformNodeBox`